src/renderer/components/output.tsx

```diff
@@ -138,7 +138,7 @@ export class Output extends React.Component<CommandsProps> {
 
     // The last 1000 lines
     const lines = output
-      .slice(Math.max(output.length - 1000, 1))
+      .slice(Math.max(output.length - 1000, 0))
```

Comment:
```suggestion
      .slice(-1000)
```

Suggested by @dsanders11